### ISSUE TITLE
Deployment error

### ISSUE THREAD

**Issue by malommey** [PatrickAlphaC/ethers-simple-storage-fcc#75]
when i try to deploy.js i get this   "at node:internal/main/run_main_module:17:47 {
  errno: -2,
  syscall: 'open',
  code: 'ENOENT',
  path: './simpleStorage_sol_simpleStorage.abi' "
}

**Comment by malommey** [comment]
when i try to deploy.js i get this "at node:internal/main/run_main_module:17:47 {
errno: -2,
syscall: 'open',
code: 'ENOENT',
path: './simpleStorage_sol_simpleStorage.abi' "
}

**Comment by rainbowginger** [comment]
I have this issue on deploying
```
JsonRpcProvider failed to startup; retry in 1s
Error: connect ECONNREFUSED 127.0.0.1:8545
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1532:16) {
  errno: -111,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 8545
}
```

**Comment by af66** [comment]
@rainbowginger me too :( did you find a solution ?

**Comment by rainbowginger** [comment]
Yes. Go the settings on right upper corner, then server, change IP to vEthernet. Refresh the new change. Use the new ip.
Issue is because ganache is in windows and wsl is in linux

**Comment by malommey** [comment]
> Yes. Go the settings on right upper corner, then server, change IP to vEthernet. Refresh the new change. Use the new ip.
> Issue is because ganache is in windows and wsl is in linux

setting on where?

**Comment by rainbowginger** [comment]
On the ganache app. Where u can see wallet addresses
